src/main/java/uk/gov/justice/hmpps/probationteams/controllers/ProbationAreaController.kt

```diff
@@ -28,6 +29,7 @@ import uk.gov.justice.hmpps.probationteams.services.SetOutcome
   value = ["probation-areas"],
   produces = [APPLICATION_JSON_VALUE],
 )
+@PreAuthorize("hasAnyRole('MAINTAIN_REF_DATA', 'SYSTEM_USER', 'ROLE_SYSTEM_USER')")
```

Comment:
Can you also remove all the `PreAuthorize` annotations from the service layer?  So we just have it at the controller level?